### ISSUE TITLE
log results to the output panel instead of the terminal [enhancement request]

### ISSUE THREAD

**Issue by Cspeisman** [firsttris/vscode-jest-runner#100]
It would awesome to have the option of logging the results of the test to the output panel instead of the terminal. A lot of times the terminal results scroll past the failed expectation because it spits out the failed block of code. It would be really useful if the user could see a formatted output of the failed expectation to the output window.

**Comment by SpencerKaiser** [comment]
I agree with the end result here but want to add another annoyance related to the terminal; whenever I open up a project, the jest terminal pops up and effectively prevents me from using ```⌘ + ` ``` to open the terminal. Instead, I need to use that command and then kill the terminal just so I can get a new terminal session started 😕

**Comment by firsttris** [comment]
there is now also a test-explorer option which uses a panel instead of terminal. re-open if you still have issues.

**Comment by Cspeisman** [comment]
thanks @firsttris - yup I've completes switched over the test-explorers now
